spv-wallet-js-client/README.md

```diff
@@ -26,7 +26,23 @@ node index.js
 
 ## Usage
 
-<!-- TODO -->
+```javascript
+import { SpvWalletClient, generateKeys } from 'spv-wallet-js-client';
+
+const server = "http://localhost:3003/v1";
+const xPriv = generateKeys().xPriv()
```

Comment:
This can be confusing. I would prefer to have an example with the xpriv string. 
 * Can we use a raw string in code? 
 * Should we create a key instance based on it, and if so, how do we do it?

Reply:
Changed